File: Leetcode_2706.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        a=sorted(prices)
        for i in range(len(a)):
            if a[0]+a[1]<=money:
                return money-(a[0]+a[1])
            else:
                return money
        

File: Leetcode_2706.java


class Solution {
    public int buyChoco(int[] prices, int money) {
        Arrays.sort(prices);  // Sort the prices in ascending order
        if (prices[0] + prices[1] <= money) {
            return money - (prices[0] + prices[1]);  // Buy two cheapest chocolates
        } else {
            return money;  // Not enough money, return full amount
        }
    }
}
